### ISSUE TITLE
Anchor - add accessibility information on ds site

### ISSUE THREAD

**Issue by jcfilben** [grommet/grommet#7447]
Using the work done in this ticket: https://github.com/grommet/hpe-design-system/issues/4382. Create a Figma draft of the accessibility information that should be added to the Anchor component page. From there devs can add it to the site.

Deliverables:
- [ ] determine which WCAG rules apply to the Anchor component
- [ ] evaluate which rules are passing or failing (may need to test with automated testing in storybook, manual keyboard testing and manual screenreader testing)
- [ ] use the template created in #4382 to create a section in the Anchor page for accessibility
